documentation/mumctl.txt

```diff
@@ -98,6 +98,19 @@ mumctl volume <user> set <volume> ::
     Set the volume of another user's incoming audio.
     1.0 is the default.
 
+mumctl messages [-i|--interactive]
+    Prints all received messages since mumd was started, or since this command last was issued,
+    whichever happens first.
+    If the interactive flag is set, mumctl will instead wait for new messages to come in and print
+    them as they come in. To exit this loop, issue a Ctrl-C.
```

Comment:
Can we send messages from the interactive view as well? If not, I think "follow" is a better name.